azure_monitor/README.rst

```diff
@@ -0,0 +1,17 @@
+Azure Monitor Exporters using OpenTelemetry
+===========================================
+
+This library provides integration with Azure Monitor using OpenTelemetry.
+
+Installation
+------------
+
+::
+
+    pip install azure-monitor-opentelemetry-exporters-python
```

Comment:
The package is called differently in setup.cfg file
opentelemetry-azure-monitor-exporters-python

Reply:
Why would one want to name a python package with `python` in the name? Do we have an npm package named `foobar-javascript` or `foobar-ecmascript`?

Reply:
Funny you mention that, we actually do have foobar-js type of names in npm, but that is a different story, I would suggest opentelemetry-azure-monitor-exporter as package name, also we need to cleanup existing one in pypi https://pypi.org/project/opentelemetry-ext-azure-monitor/

Reply:
+1 on the need to clean up existing one if we decided to go with a different name.

Reply:
`opentelemetry-azure-monitor-exporter` sounds good. Will have to clean up the existing PyPi package.